dockerfile/security/missing-user-entrypoint.yaml

```diff
@@ -4,6 +4,7 @@ rules:
     - pattern: |
         ENTRYPOINT $...VARS
     - pattern-not-inside: |
+        ...
         USER $USER
         ...
```

Comment:
Not sure if this can be helped, but starting a pattern with an ellipses is very slow! (I think we have CI checks to not allow this)

Reply:
To avoid a starting ellipses, maybe we could do something like this?

```
    - pattern: |
        ENTRYPOINT $...VARS
    - pattern-not-inside:
        USER $USER
        ...
        ENTRYPOINT $...VARS
    - pattern-not-inside:
        CMD $...VARS
        ...
        ENTRYPOINT $USER
```